docs/architecture/gameUtil.md

```diff
@@ -0,0 +1,52 @@
+# GameUtil
+
+The [GameUtil](../../src/utils/gameUtil.ts) orchestrates
```

Comment:
What is `src/utils/gameUtil.ts`? I'm guessing it's something that doesn't exist yet?

Reply:
It doesn't exist yet. See the [other PR](https://github.com/normal-devs/klicker-knight/blob/207d56c1aee1b6be7d463f569ab2be08ee1f41dd/src/utils/gameUtil.ts)

Reply:
I see how that link can be confusing lol